### PR TITLE
Move file to package codemod

### PR DESCRIPTION
**Pull request by fersilva16**
- [x] Tests
- [x] Move file to another package
- [x] Fix imports on old packages
- [x] Add a named export of the file on index.ts
- [x] Fix exports on old packages

### CODE REVIEW

File: test/dirEqual.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import fs from 'fs';
+import path from 'path';
+
+export default function dirEqual(expected: string, actual: string) {
```
Comment (sibelius) [inline test/dirEqual.ts:4]: Named export is better

**Review by sibelius — APPROVED** [review]

File: test/dirEqual.ts (hunk 1 above)
Comment (fersilva16) [reply test/dirEqual.ts:4]: Why? Is it better for auto-complete? Or there's another reason?

Comment (noghartt) [reply test/dirEqual.ts:4]: IMO some reasons to do it is:

- Follow the pattern in the codebase, the packages are using named export too.
- Is better to deal with names on imports. When you import something that was default exported you can rename it with no problem and create a misinterpretation of the code.

By the way, it's what I commented [here](https://github.com/entria/codemods/pull/22#discussion_r776297075) too.

Comment (fersilva16) [reply test/dirEqual.ts:4]: Ohhh ok, now I understand, makes sense, I'll change it, thanks!

**Review by edumaciel10 — COMMENTED** [review]
👍🏽

**Pull request merged**